packages/client/utils/draftjs/convertMobileValueToTaskContent.ts

```diff
@@ -0,0 +1,10 @@
+import {ContentState} from 'draft-js'
+import {convertStateToRaw} from './convertToTaskContent'
+
+const convertAndroidValueToTaskContent = (spacedText: string) => {
```

Comment:
Thanks for catching this @igorlesnenko, we actually built this function specifically for Android devices since these were the only devices experiencing this. Hence, we have updated the file name accordingly.